Makefile

```diff
@@ -1,4 +1,4 @@
-GO_DOCKER_RUN = docker run --rm -v $(shell pwd)/cmd/sync:/go/src/github.com/nginxinc/nginx-asg-sync/cmd/sync -v $(shell pwd)/build:/build -w /go/src/github.com/nginxinc/nginx-asg-sync/cmd/sync
```

Comment:
removing `-v $(shell pwd)/build:/build ` breaks the build

Reply:
See the other PR @pleshakov and review that only please